### PR TITLE
Validate input values with JSON Schema + docs update

### LINKED ISSUE #31: Add support for Schema Files to validate values.yaml
https://helm.sh/docs/topics/charts/#schema-files

### PR DESCRIPTION
**Pull request by ekampf**
#### What this PR does / why we need it:
Add JSON Schema file to validate input `values.yaml` values

#### Which issue this PR fixes if any
Fixes #31 

#### Special notes for your reviewer:
In this PR:
- Added `stable/connector/values.schema.json` - Helm will automatically use this to validate given values
- Added `stable/connector/ci/*-values.yaml` - Our CI's `helm-lint` step will automatically `helm lint` all these different values and make sure they work
- Added docs to `stable/connector/values.yaml`

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

File: stable/connector/templates/deployment.yaml [hunk 1]
```diff
@@ -49,7 +49,7 @@ spec:
                 optional: false
           env:
             - name: TWINGATE_URL
-              value: "https://{{ required "Network name required" .Values.connector.network }}.{{ .Values.connector.url }}"
+              value: "https://{{ required "Network name required" .Values.connector.network }}.{{ .Values.connector.url | default "twingate.com" }}"
```
Comment (liorr) [inline stable/connector/templates/deployment.yaml:52]: I didn't see a test where we test overriding the default value

File: stable/connector/values.yaml [hunk 2]
```diff
@@ -2,48 +2,71 @@
...
-# (https://kubernetes.io/docs/tasks/administer-cluster/sysctl-cluster/#enabling-unsafe-sysctls)
+## Enable the Linux kernel's net.ipv4.ping_group_range parameter to allow ping connector.
+## Use only if you enable this sysctls in your cluster (disabled by default)
+## Or have Kubernetes master higher than 1.18
+## (https://kubernetes.io/docs/tasks/administer-cluster/sysctl-cluster/#enabling-unsafe-sysctls)
 icmpSupport:
   enabled: false
 
-connector:
-  logLevel: "error"
```
Comment (liorr) [inline stable/connector/values.yaml:43]: Where do we set the default for logLevel?

**Review by liorr — CHANGES_REQUESTED** [review]
Please check my questions and comments.

File: stable/connector/values.yaml (hunk 2 above)
Comment (ekampf) [reply stable/connector/values.yaml:43]: Also I modified `test/golden/default-values.yaml` to test this...  (removed it but the generated template is the same)

**Review by liorr — APPROVED** [review]
LGTM

File: stable/connector/README.md [hunk 3]
```diff
@@ -79,11 +79,10 @@ The following table lists the configurable parameters of the Twingate chart and
 | `affinity`                              | Map of node/pod affinities                                                  | `{}` (The value is evaluated as a template)             |
 | `nodeSelector`                          | node labels for pod assignment                                              | `{}` (The value is evaluated as a template)             |
 | `tolerations`                           | Tolerations for pod assignment                                              | `[]` (The value is evaluated as a template)             |
-| `resources`                             | Resrouce limitations                                                        | `{}` (The value is evaluated as a template)             |
+| `resources`                             | Resource requests and limits                                                | `{}` (The value is evaluated as a template)             |
```
Comment (ekampf) [inline stable/connector/README.md:82]: Fixed typo

**Pull request merged**